src/components/GraphFilter/DurationDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { DropdownButton, MenuItem } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+type DurationDropdownProps = {
+  disabled: boolean;
+  initialDuration: string;
+  onClick: PropTypes.func;
+};
+
+type DurationDropdownState = {
+  current: string;
+};
+
+export class DurationDropdown extends React.Component<DurationDropdownProps, DurationDropdownState> {
+  static DurationIntervals = [
+    ['60', '1 minute'],
```

Comment:
Shouldn't the first field be an integer not string? It represents the actual value in seconds.